docs/elasticsearch.rst

```diff
@@ -1,67 +1,52 @@
 ======
 Search
 ======
+Kuma uses Elasticsearch_ to power its on-site search.
 
-Kuma uses `Elasticsearch <http://www.elasticsearch.org>`_ to power its
-on-site search facility.
+.. _Elasticsearch: https://www.elastic.co/products/elasticsearch
 
-Elasticsearch search gives us a number of advantages over MySQL's full-text
-search or Google's site search.
+Installed Version
+=================
+Elasticsearch `releases new versions often`_, and Kuma is slow to upgrade.  The
+Vagrant and Docker environments use 1.7, but production is older.
 
-* Much faster than MySQL.
-  * And reduces load on MySQL.
-* We have total control over what results look like.
-* We can adjust searches with non-visible content.
-* We don't rely on Google reindexing the site.
-* We can fine-tune the algorithm ourselves.
+The Kuma search tests use the configured Elasticsearch, and can be run to
+confirm the engine works::
```

Comment:
You might want to either add a note that this command should be run inside a vagrant VM or a docker container, or remove that explanation for the `./manage.py reindex` instructions below and assume that the person reading this part of the docs already understands how to run python commands in the appropriate environment.